src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class SortCommand extends Command{
```

Comment:
Do look out for the missing whitespace between `Command` and `{`, and the javadoc comment for the class.